### PR TITLE
INGK-743 Load dictionary by default in the virtual drive

### PR DESCRIPTION
**Pull request by pzinemanas**
### Description

This PR loads a default dictionary in the virtual drive.

Fixes # INGK-743

### Type of change

Please add a description and delete options that are not relevant.

- [x] Load dictionary by default in the virtual drive
- [x] Move default dictionary from tests to module folder.

### Tests
- [x] Add new unit tests if it applies.
- [x] Run tests.

### Documentation

Please update the documentation.

- [x] Update docstrings of every function, method or class that change.
- [x] Build documentation locally to verify changes.
- [x] Add the changes at the `[Unreleased]` section of the [CHANGELOG](CHANGELOG.md).

### Code formatting

- [x] Use black package to format the code: `black -l 100 ingenialink tests`. It is recommended to configure the code editor to automatically format the code using black with a max length line of 100.

### CODE REVIEW

**Review by WoodElv — APPROVED** [review]

File: tests/test_dictionary.py [hunk 1]
```diff
@@ -1,32 +1,30 @@
...
 
 import pytest
-from os.path import join as join_path
 
 from ingenialink.canopen.dictionary import CanopenDictionary
 from ingenialink.ethernet.dictionary import EthernetDictionary
 
-
 path_resources = "./tests/resources/"
+path_to_dictionary = "./ingenialink/virtual/resources/virtual_drive.xdf"
```
Comment (GullinBustin) [inline tests/test_dictionary.py:13]: Use Uppercase to constants

File: ingenialink/virtual/virtual_drive.py [hunk 2]
```diff
@@ -321,26 +323,27 @@ class VirtualDrive(Thread):
...
-    def __init__(
-        self, ip: str, port: int, dictionary_path: str = "./tests/resources/virtual_drive.xdf"
-    ) -> None:
+    def __init__(self, ip: str, port: int, dictionary_path: Optional[str] = None) -> None:
         super(VirtualDrive, self).__init__()
         self.ip = ip
         self.port = port
-        self.dictionary_path = dictionary_path
+        default_dictionary = os.path.join(
+            pathlib.Path(__file__).parent.resolve(), "./resources/virtual_drive.xdf"
```
Comment (GullinBustin) [inline ingenialink/virtual/virtual_drive.py:339]: Use constants instead literal

**Review by GullinBustin — CHANGES_REQUESTED** [review]

File: ingenialink/virtual/virtual_drive.py (hunk 2 above)
Comment (pzinemanas) [reply ingenialink/virtual/virtual_drive.py]: Done!

File: tests/test_dictionary.py (hunk 1 above)
Comment (pzinemanas) [reply tests/test_dictionary.py]: Done!

**Pull request merged**